<launch>

	<node pkg="rosserial_python" type="serial_node.py" name="rosserial" output="screen">
	  <param name="port" value="/dev/ttyACM0" />
	  <param name="baud" value="57600" />
	</node>

	<node pkg="roscar" type="record_video" name="record_video"></node> -->

	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video0" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<param name="pixel_format" value="mjpeg" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
	</node>
	<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
		<remap from="image" to="stored_image"/>
		<param name="autosize" value="false" />
	</node>
  
</launch>
